firebase-plugin/write-to-db.ts

```diff
@@ -0,0 +1,5 @@
+import firebase from "firebase";
+
+export const writeToDB = (ref: firebase.database.Reference, report: any) => {
```

Comment:
why you're not using `export default` here since it exports only one specific function?

Reply:
I usually avoid using export default to improve naming consistency, so the consumer won't be able to change the name when importing without being explicit and using `import { as }`.

Reply:
hmm got it, sounds clever also, but I am still wondering whether this would actually be a problem. Most of the time we can catch this in simple code review processes. 

Since this file has the same name as the exported function, and only 1 function, I would really expect for it to be a `default` import, mainly for semantic reasons.

If you still want to stick with it, I would recommend at least giving the file a more generic name then, something like `firebase-actions.ts`  then you can just import using `import { writeToDB } from '../firebase-actions'`

We keep it semanticaly correct, and we can add more actions.